js/glsl/mesh-vertex.glsl

```diff
@@ -23,11 +23,9 @@ attribute vec3 position_previous;
     varying vec2 vertex_uv;
 #endif
 
-attribute vec4 color;
+attribute vec4 color_current;
```

Comment:
Was there a need to rename this? Does it collide with threejs' naming?

Reply:
There already exists a variable named 'color' in the shader code from three.js. I don't remember exactly in which include, but yes, it was necessary. 
I will go into more depth when we being refactoring/fixing the code.

Reply:
**uniform vec3 color** is defined in shadow_frag.glsl and again in removed line **attribute vec4 color**